.travis.yml

```diff
@@ -5,16 +5,21 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - ST2_REPO_PATH=/tmp/st2
+
 before_install:
   - sudo mkdir -p /var/log/opensds
   - sudo touch /var/log/opensds/orchestration.log
   - sudo chmod 666 /var/log/opensds/orchestration.log
+  - git clone https://github.com/StackStorm/st2.git /tmp/st2
 
 install:
   - pip install tox codecov
 
 script:
   - tox
+  - $ST2_REPO_PATH/st2common/bin/st2-run-pack-tests -p contrib/st2/opensds
 
```

Comment:
please andd script in to tox, otherwise the codecov will not caculate the coverage of unittest